src/common/com/intellij/plugins/haxe/lang/psi/impl/HaxePsiFieldImpl.java

```diff
@@ -62,25 +64,16 @@ public String getName() {
   }
 
   @Override
-  @NotNull
+  @Nullable
   public HaxeComponentName getComponentName() {
-    return new HaxeComponentNameImpl(getNode());
-    ////this.getName();
-    //return findNotNullChildByClass(HaxeComponentName.class);
+    return null;
```

Comment:
So, the answer is to strip the implementation?  The function is obviously used (or you wouldn't have gotten the NPE), so some other functionality depends upon it.